### ISSUE TITLE
Make the debugging port an argument to `debugOptions`

### ISSUE THREAD

**Issue by earldouglas** [earldouglas/xsbt-web-plugin#326]
The debugging port is buried within a string in `debugOptions`:

```scala
debugOptions := Seq( "-Xdebug"
                   , "-Xrunjdwp:transport=dt_socket,address=8888,server=y,suspend=n"
                   )
```

This will pose a problem for #191, which will lead to (debugging) port conflicts between scaled instances.

Add a new setting, `debugPort`, and use it to set `debugOptions`.

**Comment by earldouglas** [comment]
Looks like it should be a string, not an integer: https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/introclientissues005.html

**Comment by earldouglas** [comment]
Fixed by #332.
